pillarbox-demo-shared/src/main/java/ch/srgssr/pillarbox/demo/shared/ui/integrationLayer/ContentList.kt

```diff
@@ -21,15 +21,23 @@ sealed interface ContentList {
         val bu: Bu
     }
 
+    interface ContentListFactory<T : ContentList> {
```

Comment:
```suggestion
    interface ContentListFactory<T: ContentList> {
```

Reply:
In Kotlin, we put a whitespace before and after the colon for inheritance (here, we say that `T` is a generic type that extends `ContentList`)